### DISCUSSION TITLE
Problem setting up without using .env file.

### DISCUSSION THREAD

**Discussion by MichalisDBA** [Kovah/LinkAce#402] [Support] [answered]
Hello.

I try to setup the docker container with this docker compose guide
https://www.linkace.org/docs/v1/setup/setup-with-docker/advanced-configuration/
Using Docker environment variables instead of the .env file

I have removed from the docker compose the
- ./.env:/app/.env

When i run the container and press Check Requirements button all settings are checked - green except the 
 .env file is present and writable
and it does not let me continue the installation.

I don't use the .env file as i use the guide above without the .env file to setup the linkace container.

**Comment by Kovah** [comment, marked as answer]
To get this solved, run the commands from https://www.linkace.org/docs/v1/setup/setup-with-postgresql/ to set up the database and create a user.

I will update the wiki accordingly.

**Reply by MichalisDBA** [reply]
We use MySQL not Postgresql. Does the same steps apply?

**Reply by Kovah** [reply]
Yes, it's just about the commands itself. If you correctly configured the .env file you should be good to go.

**Reply by MichalisDBA** [reply]
Thanks i did that. The database was created and after that i go to login screen but i haven't setup a user account and now asks me for login credentials.

**Reply by Kovah** [reply]
You have to run the command to register a user too. (`docker exec -it linkace_app_1 php artisan registeruser`)

**Reply by MichalisDBA** [reply]
Thanks problem solved.
